### ISSUE TITLE
[Research]Add the candidates that are associated with the JF committee participants

### ISSUE THREAD

**Issue by fec-jli** [fecgov/openFEC#4861]
### What we’re after
The API user can get the candidates that are associated with the JF committee participants 
from endpoints(CommitteeHistoryView):
 **'/committee/<string:committee_id>/history/',**
**'/committee/<string:committee_id>/history/<int:cycle>/'**

### Action item(s)
- [x] Check if the needed data available in database table. 
- [ ] How to modify ofec_committee_history_mv
- [ ] Open up any follow up tickets if applicable 

### Completion Criteria 
-[ ] Research is documented 

### References/resources/technical considerations
1)F1: https://docquery.fec.gov/pdf/926/202007019244226926/202007019244226926.pdf

2)Slack link: https://fecgov.slack.com/archives/C3XM8V6LE/p1620928013216300
<img width="494" alt="Screen Shot 2021-05-13 at 4 10 35 PM" src="https://user-images.githubusercontent.com/24395751/118325345-57416480-b4d1-11eb-8815-8ae6b23aae88.png">

3)api url:
https://api.open.fec.gov/v1/committee/C00750166/history/2020/?api_key=DEMO_KEY&sort_null_only=false&sort=-cycle&sort_nulls_last=false&page=1&sort_hide_null=false&per_page=20&election_full=true

https://api.open.fec.gov/v1/committee/C00639591/history/2020/?api_key=DEMO_KEY&sort_null_only=false&sort=-cycle&sort_nulls_last=false&page=1&sort_hide_null=false&per_page=20&election_full=true

https://api.open.fec.gov/v1/committee/C00639591/candidates/history/2020/?api_key=DEMO_KEY&page=1&sort_hide_null=false&sort_null_only=false&sort=-two_year_period&per_page=20&election_full=true&sort_nulls_last=false

**Comment by fec-jli** [comment]
apiinfo email from a user:
"I have a question about your API. I'm trying to find a way to programmatically connect candidates to their fundraising Committees.

For example, according to a filed F1 form, The Squad Victory Fund with FEC Committee ID # C00750166 is associated with a number of other Committees for individual candidates, and other causes. However, when searching through the API, there seems to be no such association between the Victory Fund and the individual candidates (i.e. Alexandria Ocasio-Cortez, Candidate ID # H8NY15148) when looking at the data.

Perhaps I am reading the form wrong and that this Committee is in fact the incorrect one? I'm confused because according to how I am interpreting the Form, this is the Committee that processes donations for the listed sub-Committees.

In short, I am looking for a way to match individual candidates with the Committees that process their donation funds via your API, and am wondering if there is a standardized method for achieving this. Ultimately my goal in theory is to be able to send donations directly to the affiliated campaigns of candidates by inputting such candidate (by ID#, etc).

Of course in doing so, compliance with FEC laws and regulations is imperative."

**Comment by hcaofec** [comment]
The id and name of JF committee participants are stored in Form_1s.  Currently committee APIs return its JFC id and name by joining Form_1s.  If additional data such as candidates that are associated to the JF committee participants are to be added to the output,  additional join to committee table is required.  Instead of refactoring ofec_committee_history_mv,  a separate table is recommended.  This table includes JFC, its participants, candidates if there is any.  

For now, user can use the committee API filter by the id of JFC participants to find the candidates associated with it.

No following up tickets for now.
